src/alto-ui/SideNav/SideNav.scss

```diff
@@ -348,7 +348,7 @@ $nav-btn-hover-text-dark: $white;
 .sidenav__panel {
   position: absolute;
   background: $white;
-  box-shadow: 1px 0 0 $coolgrey-20;
+  box-shadow: inset -1px 0 0 $coolgrey-20;
```

Comment:
Let's find different way to hide this "border"

Reply:
I changed it